recipes/raylib/config.yml

```
@@ -1,3 +1,3 @@
 versions:
-  "3.5.0":
+  "4.0.0":
```

Do not remove 3.5.0 version
```suggestion
    folder: "all"
  "4.0.0":
```